cmd/install/addon/cmd.go

```diff
@@ -197,61 +195,36 @@ func run(cmd *cobra.Command, argv []string) {
 		parameters.Each(func(param *cmv1.AddOnParameter) bool {
 			var val string
 			var hasVal bool
+			input := &interactive.Input{
+				Question: param.Name(),
+				Help:     fmt.Sprintf("%s: %s", param.ID(), param.Description()),
+				Required: param.Required(),
+			}
+			options, ok := param.GetOptions()
+			if ok {
+				input.Options = []string{}
+				for _, opt := range options {
+					input.Options = append(input.Options, opt.Value())
+				}
+			}
```

Comment:
is this part missing from the edit command on purpose?

Reply:
No, not on purpose. I think I lost it when I refactored the function. Adding it in. 👍